src/app/detail/ui/Comments.tsx

```diff
@@ -15,7 +16,12 @@ export const Comments = ({ comment, createComment, deleteComment, task_id }: Pro
   return (
     <Stack direction="row" columnGap={2} position="relative">
       <VerticalLine />
-      <Avatar alt="user" src={''} sx={{ width: '25px', height: '25px', marginTop: '5px' }} />
+      <AvatarWithInitials
+        altName={comment?.initiator?.givenName}
+        alt="user"
+        src={''}
+        sx={{ width: '25px', height: '25px', marginTop: '5px' }}
+      />
```

Comment:
I think this component name is misleading. `AvatarWithInitials` can be anything. Can you please specify what are the initials? Better to change the component name IMO.